app/controllers/mission.rb

```diff
@@ -0,0 +1,89 @@
+get '/mission/new' do
+  #this should only provide currently avaliable ppl,
+  #off-line until the 'prevent double booking' validation is added
+  @personnel = Soldier.all
+  #as w/ ppl
+  @trucks = Truck.all
+  @trailers = Trailer.all
+  erb :'planning/new_mission'
+end
+
+post '/mission' do
+  @mission = Mission.create_new(params)
+  redirect '/'
+end
+
+get '/mission/forms/:type' do
```

Comment:
this isn't very RESTful. not super suer what the goal is of this route. consider what you are accomplishing by hitting this route, then determine what the path name should be